### PR TITLE
feat: added homepage

### PR DESCRIPTION
**Pull request by shrushti2000**
### Added homepage

ezgif com-gif-maker (9)

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,164 @@
+.app-name-1{
+  color: var(--white);
+  font-size: 2rem;
+}
+.app-name-2{
+  color:var(--primary-bg-color);
+  font-size: 2rem;
+}
```
Comment (xii-vi) [inline homepage.css:8]: _You could try some generic name._

**Review by xii-vi — APPROVED** [review]
Just a tiny bit of suggestion mentioned in a comment, the rest is great. You can merge.
Great Work on the UI.

File: homepage.css [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+.overlay {
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 2;
```
Comment (haarsh24) [inline homepage.css:44]: You can also declare background color

File: homepage.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+.sub-heading {
+  font-size: 3rem;
+  color: var(--primary-text-color);
+  text-align: center;
+}
+
+.btn-shop-now {
+  position: absolute;
+  color: white;
```
Comment (haarsh24) [inline homepage.css]: Don't use color directly

**Review by haarsh24 — APPROVED** [review]

**Pull request merged**